### PR TITLE
Delete code that supports marathon

### PR DESCRIPTION
**Pull request by EvanKrall**

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -2139,27 +2139,27 @@ def test_deploy_blacklist_to_constraints():
...
         "paasta_tools.utils.get_service_instance_list",
         autospec=True,
-        side_effect=[mock_marathon_instances],
+        side_effect=[mock_kubernetes_instances],
     ):
         assert (
             utils.validate_service_instance(
                 my_service, my_instance, fake_cluster, fake_soa_dir
             )
-            == "marathon"
```
Comment (nemacysts) [inline tests/test_utils.py:2157]: i'm sorta confused here: how come this isn't `kubernetes`? seems like an interesting choice for us to have had a test that just tested what the first item in a tuple was

**Review by nemacysts — COMMENTED** [review]
generally looks good (apart from needing to fix https://github.com/Yelp/paasta/pull/3109/files/05df09be25fccef7b1e32154099c87d6275ff9dc#diff-33c13e0b177bacd2f02e29bcb8aea5b49e7ce34901fd8f41fefb65defba1bd33R152) - the only thing that worries me is that some users internally will reach into our venv and use our internals and we've removed/moved a decent number of things in this PR. additionally: we should figure out how many people are using us as a library (e.g., nerve-tools: https://github.com/Yelp/nerve-tools/search?l=Python&q=paasta)

File: example_cluster/example-cluster-test.sh [hunk 2]
```diff
@@ -22,6 +22,5 @@ cd dockercloud-hello-world
 try_and_retry /work/example_cluster/tests/check-metastatus.sh
 try_and_retry /work/example_cluster/tests/check-status.sh
 try_and_retry /work/example_cluster/tests/check-api.sh
-paasta stop -s hello-world -c testcluster -i main
 paasta stop -s hello-world -c testcluster -i remote
```
[the remark this answers is not in the document]
Comment (EvanKrall) [reply example_cluster/example-cluster-test.sh:25]: it's adhoc: https://github.com/Yelp/paasta/blob/fcba5cd60/example_cluster/example-services/hello-world/adhoc-testcluster.yaml

**Comment by EvanKrall** [comment]
Closing in favor of https://github.com/Yelp/paasta/pull/3310; we can delete the other files in future PRs.

**Pull request closed without merging**